### PR TITLE
[65] Create & Style the customer signup page using CSS 

### PR DESCRIPTION
**Pull request by cherishebojudah**
## Task: `[65] Create & Style the customer signup page using CSS
Sign-up page mobile 1
Sign-up page mobile 2
Sign-up page tablet 1
Sign-up page tablet 2
Sign-up page web 1
Sign-up page web 2

`
- `created and designed sign-up page for ecommerce website`

### CODE REVIEW

**Comment by adaugochi** [comment]
Beautiful. well done @cherishebojudah Kindly fix cofactor issue

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done so far
Kindly place your signUp.html folder inside `html-forms` folder

File: submissions/ebojudahCherish/css/signUp.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/ebojudahCherish/css/signUp.html:1]: Kindly place this file inside `html-forms` folder. 
image

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — COMMENTED** [review]
You currently have two signup.html files. 1 in css folder and the other in html-forms folders. Kindly delete the one in css folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
Rename your `styles.css` to `signup.css`

**Comment by cherishebojudah** [comment]
> Rename your `styles.css` to `signup.css`

@adaugochi about this, this file holds all the styling for all my pages, so it might not be ideal for me to name it 'signup.css'. Thanks for all your feedback.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**